lib/parse_config.js

```diff
@@ -30,7 +31,12 @@ function parseObjRepo(repo) {
       } else {
         userToken = configToken;
       }
-      repoUrl.username = userToken;
+      if (token_name && repoUrl.host === 'e.coding.net') {
```

Comment:
```suggestion
      if (token_name && repoUrl.host.includes('coding.net') {
```

Coding has its git repo url changed too many times.